org.opensurge2d.OpenSurge.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: org.opensurge2d.OpenSurge
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+default-branch: stable
+sdk: org.freedesktop.Sdk
+command: opensurge
+rename-icon: opensurge
+rename-desktop-file: opensurge.desktop
+rename-appdata-file: opensurge.appdata.xml
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=home:ro
```

Comment:
Not sure why you need home access here

Reply:
Open Surge is a game engine and a game. Users can download new games from the community and install them on the engine. These downloads are usually stored in the home. The project needs home access to be able to run games made by the community.

Reply:
Can you make it `xdg-download` instead? Users can override these permissions after install.

Reply:
I can, but it's not ideal. If the file is stored outside xdg-download for some reason (like in $HOME), the engine will not be able to load it and users may not know why. This may cause difficulty: such limitation didn't exist and is not explained anywhere.